package.json

```diff
@@ -3,15 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/react-router-dom": "^5.1.5",
+    "express": "^4.17.1",
     "firebase": "^7.14.2",
     "firebase-admin": "^8.10.0",
     "firebase-tools": "^8.0.1",
+    "node": "^14.4.0",
+    "nodemon": "^2.0.4",
```

Comment:
what are these two dependencies for? it's a little suspicious that a `node` application would require itself as a dependency. In addition, `react-scripts` should take care of hot-reloading so there shouldn't be a need for `nodemon`

Reply:
The node is for the server that I'm currently working on but not yet ready for but still in testing, its required for any server side task using node and express.